app/assets/stylesheets/admin/dashboard/_dashboard.scss

```diff
@@ -9,34 +9,40 @@
     .section {
       padding: 1em;
 
+      header {
+        display: flex;
```

Comment:
Fun. I need to learn this if I'm going to be code reviewing it! Perhaps you can teach me...